### ISSUE TITLE
Support v1 being a branch

### ISSUE THREAD

**Issue by eregon** [nektos/act#234]
Just like GitHub Actions supports using `my/action@branch`.

Same as https://github.com/nektos/act/issues/101#issuecomment-628281226 but a new issue as that one was closed.

**Comment by eregon** [comment]
Not stale (AFAIK).

**Comment by eregon** [comment]
BTW I noticed the template for JS actions recommends a `v1` branch:
https://github.com/actions/javascript-action#create-a-release-branch

**Comment by eregon** [comment]
Not stale (AFAIK).

**Comment by jeffdoolittle** [comment]
I'd really like to see this working as well.

**Comment by Sheeri** [comment]
+1

**Comment by tadman** [comment]
Keep bumping into this when trying to use a common action:

```yaml
      - name: Setup Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: '2.7'
```

Here `v1` is a branch, not a tag.

**Comment by cristianmtr** [comment]
Is there at least a workaround for this?

I am trying

```
jobs:
  some-task:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
        with:
          submodules: true
      - uses: actions/setup-python@v1
        with:
          python-version: 3.7
      - run: |
          echo "works!!!!"
...
```

and get `Unable to resolve v1: reference not found`

**Comment by eregon** [comment]
Not stale.

**Comment by lufia** [comment]
@eregon I think this has probably fixed at current main branch sources. Have a try.

**Comment by TheCleric** [comment]
Not stale.

**Comment by TheCleric** [comment]
Not stale.

**Comment by catthehacker** [comment]
This should have been fixed already by #433.
